### DISCUSSION TITLE
How to use board not yet supported? STM32F031C6TX

### DISCUSSION THREAD

**Discussion by TCodingB** [stm32duino/Arduino_Core_STM32#1838] [Q&A] [answered]
Hello,

I have a custom board with STM32F031C6TX mcu, and would like to test SimpleFOC library. Since this current board/mcu isn't yet supported in STM32 2.3 (but I see it on a list of version 2.4) I am looking for other solutions. What would be the best approach to solve this? Maybe another IDE, how can I add support/add custom mcu?

Edit: When is the release of the 2.4 version?

Thank you in advance for your answers,

Tadej

**Comment by fpistm** [comment, marked as answer]
Hi,

you can simply copy the required files from the main branch and declare it to the boards.txt:

https://github.com/stm32duino/Arduino_Core_STM32/tree/main/variants/STM32F0xx/F031C(4-6)T
https://github.com/stm32duino/Arduino_Core_STM32/blob/5818d75ab7561f66d9a234ab7e16c9b51a6bbbad/boards.txt#L1139-L1145

2.4.0 will be release in the coming weeks.

**Comment by TCodingB** [comment]
Solved! 🗡️ Thank you very much.
